src/ValueFormatters/QuantityFormatter.php

```diff
@@ -191,13 +191,13 @@ private function getRoundingExponent( QuantityValue $quantity ) {
 	}
 
 	/**
-	 * @param DecimalValue $margin
+	 * @param DecimalValue|null $margin
 	 * @param int $roundingExponent
 	 *
 	 * @return string|null Text
 	 */
-	private function formatMargin( DecimalValue $margin, $roundingExponent ) {
-		if ( $this->options->getOption( self::OPT_SHOW_UNCERTAINTY_MARGIN ) ) {
+	private function formatMargin( DecimalValue $margin = null, $roundingExponent ) {
+		if ( $margin !== null && $this->options->getOption( self::OPT_SHOW_UNCERTAINTY_MARGIN ) ) {
```

Comment:
for backwards compat, we need multiple values now: NEVER, IF_KNOWN, and IF_NOT_ZERO. IF_KNOWN is what we really want, IF_NOT_ZERO can be used to emulate the current behavior.